prism/src/main/java/org/apache/falcon/plugin/GraphiteNotificationPlugin.java

```diff
@@ -45,33 +45,36 @@ public void monitor(ResourceMessage message) {
         MetricNotificationService metricNotificationService =
                 Services.get().getService(MetricNotificationService.SERVICE_NAME);
         try {
-            String entityType = message.getDimensions().get("entity-type");
-            String entityName = message.getDimensions().get("entity-name");
+            String entityType = StringUtils.isNotBlank(message.getDimensions().get("entityType"))
+                    ? message.getDimensions().get("entityType") :message.getDimensions().get("entity-type");
+            String entityName = StringUtils.isNotBlank(message.getDimensions().get("entityName"))
+                    ? message.getDimensions().get("entityName") :message.getDimensions().get("entity-name");
             String prefix = StartupProperties.get().getProperty("falcon.graphite.prefix");
-            if (entityType.equals(EntityType.PROCESS.name())) {
+            String separator = ".";
+            LOG.debug("message:" + message.getAction());
+            if (entityType.equalsIgnoreCase(EntityType.PROCESS.name())) {
                 Entity entity = ConfigurationStore.get().get(EntityType.PROCESS, entityName);
                 Process process = (Process) entity;
                 String pipeline =  StringUtils.isNotBlank(process.getPipelines()) ? process.getPipelines() : "default";
 
-
                 if ((message.getAction().equals("wf-instance-succeeded"))) {
                     Long timeTaken =  message.getExecutionTime() / 1000000000;
-                    String metricsName = prefix + message.getDimensions().get("cluster") + pipeline
-                            + ".GENERATE." + entityName + ".processing_time";
+                    String metricsName = prefix + separator +message.getDimensions().get("cluster") + separator
```

Comment:
Minor nit : Not having space between + and message might cause checkstyle violation. You used separator between prefix and message, but did not use it for ".GENERATE.", IMHO the separator var might be overkill. StringBuilder() might be cleaner way of appending strings instead. 

+1 for the actual code fix.